src/main/MQ2WindowInspector.cpp

```diff
@@ -1920,7 +1920,7 @@ class ImGuiWindowStringEditor
 					Zep::ZepBuffer* buffer = m_textEditor->GetEditor().InitWithText("", sv);
 				}
 
-				m_textEditor->Render("##StringEditor", ImVec2(0, ImGui::GetContentRegionAvail().y - 26));
```

Comment:
removing this reduces the functionality of the zep editor -- it's not something that was used but keep in mind that now you need a _separate_ editor for each render whereas before you could have a text editor maintain its own state with multiple render targets

Reply:
I'm not sure what you mean by "multiple render targets" in this instance. Are you saying render the same editor multiple times?

Reply:
on one hand, we have the id stored on the widget so we don't have to provide it every frame.

on the other hand, if we want to change the id, its a bit more complicated. Do we ever have a case where we'd want to change the id of a widget we already created?

I can't really think of a practical reason why we'd want to change the id, or render twice.

Reply:
I can see a situation where we'd want to have multiple subwindows of text without wanting to instantiate an editor for each so you could track an overall editor state. if that's not important, then it's not

Reply:
no, that is not viable. Replacing the text in the editor is prohibitively expensive.

Reply:
if we did subwindows that'd be supported directly by the editor.